R/as_polars_series.R

```diff
@@ -182,10 +182,7 @@ as_polars_series <- function(x, name = NULL, ...) {
 #' @rdname as_polars_series
 #' @export
 as_polars_series.default <- function(x, name = NULL, ...) {
-  abort(
-    paste0("Unsupported class for `as_polars_series()`: ", toString(class(x))),
-    call = parent.frame()
-  )
+  abort(paste0("Unsupported class for `as_polars_series()`: ", toString(class(x))))
```

Comment:
Can we use `obj_type_friendly()`?

Reply:
Good point. I did not know that I could use that function because I had been using `toString()` since before I added the standalone file.